templates/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- start fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end fonts -->
+    <link rel="stylesheet" href="../static/css/index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- start nav -->
+    <nav class="nav" id="nav">
+      <div class="container">
+        <div class="nav__container__parnav">
+          <div class="nav__container__parnav__brand">
+            <a href="#">My Logo</a>
```

Comment:
Kindly update the `font-size` and `color` of logo to match the Figma design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/191697946-a2d1eecf-80e4-44b4-a195-d01f5fa53703.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/191697818-c9991132-0e5b-4a79-9267-64ba1614106e.png)